database/schema.sql

```diff
@@ -0,0 +1,49 @@
+DROP DATABASE IF EXISTS mvps;
```

Comment:
The database should be seeded with knexjs. refer to knew js documentation. not MySQL.